src/Tree/index.tsx

```diff
@@ -312,6 +318,23 @@ class Tree extends React.Component<TreeProps, TreeState> {
     }
   };
 
+  handleAddChildrenToNode = (nodeId: string, childrenData: RawNodeDatum[]) => {
+    const data = clone(this.state.data);
+    const matches = this.findNodesById(nodeId, data, []);
+
+    if (matches.length > 0) {
+      const targetNodeDatum = matches[0];
+
+      const depth = targetNodeDatum.__rd3t.depth;
+      const formattedChildrens = clone(childrenData).map((node: RawNodeDatum) =>
```

Comment:
this should be singular `formattedChildren`

Reply:
I updated the main branch with this change. Thanks!